project1.css

```diff
@@ -38,9 +41,44 @@
 
 .intro-page {
 	font-family: Arial;
-	padding-left: 40px;
-	color: white;
+	padding-left: 75px;
+	padding-top: 20px;
+	color: black;
 	background-color: #fc7b03;
 	background-image: linear-gradient(to bottom right, #fc7b03, #000000);
+	width: 100%;
+	height: 100%;
 	}
-}
\ No newline at end of file
+	
+button {
+	background: transparent;
+	border: none;
+}
```

Comment:
I think this is just for the icons - it would probably be better to give them a class and style them that way so that if you end up having other `<button>`s in the future, they won't be affected by this styling automatically